validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -348,9 +347,6 @@ def validate_block(self, blkw, consensus, chain_head=None, chain=None):
             blkw.status = BlockStatus.Valid
             return True
 
-        except ChainHeadUpdated:
```

Comment:
Is this never raised? It's not clear why this is a linting-only change.

Reply:
Catching the exception only to immediately raise it again has not affect and causes the linter to break.

Reply:
Isn't the intent of this line to propogate it up (skip the broad except in this case)?  After this, won't ChainHeadUpdated be caught by the broad except?

Reply:
I think it would work to change it to
```
except ChainHeadUpdated as e:
    raise e
```
Leaving it as-is crashes pylint due to a bug in the release version.